apps/app_manager.py

```diff
@@ -1,11 +1,15 @@
-import traceback
 import importlib
 import os
+import traceback
+
+from apps import zero_app
```

Comment:
I suggest putting the ZeroApp class into apps/__init__.py , so that you can simply do "from apps import ZeroApp"